### DISCUSSION TITLE
Getting console error and Livewire stopped working

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [livewire/livewire#6879] [1. Help]
All of a sudden I start getting this error on the console

```
Uncaught SyntaxError: Invalid or unexpected token (at livewire.js?id=f41737f6:7802:1)
```

![CleanShot 2023-09-21 at 03 32 20](https://github.com/livewire/livewire/assets/1639302/5722d180-237d-4261-a401-7b94da27a63f)

Any ideas on what may be causing the issue? I have tried to remove the vendor folder and recreate it. No luck. Anybody else are facing or had this issue?

**Comment by joshhanley** [comment]
@abishekrsrikaanth can you paste a screenshot of the whole error and stack trace? Also any code you think might be triggering it? Thanks!

**Comment by abishekrsrikaanth** [comment]
@joshhanley there isn't a stack trace.

![CleanShot 2023-09-21 at 04 30 43](https://github.com/livewire/livewire/assets/1639302/e74bf428-46b1-4f2f-9e26-2c142ed88611)

I tried to disable auto injection of scripts and tried to load the livewire scripts manually and still no luck. I get this error when I try to load a livewire component either using `@livewire` or `<livewire:` tag. I tried to remove all packages both npm and composer to see if one of those was causing the issue. No luck with that. Removed both `vendor` and `node_modules` and re-installed everything and still no luck. Both Livewire and Alpine aren't loaded on the page. Any Livewire actions causes console error `Livewire is not defined`

**Reply by joshhanley** [reply]
@abishekrsrikaanth ok thanks. Any chance you can make a small example repo that demonstrates the issue, as I'm unable to replicate it. Thanks!

**Reply by abishekrsrikaanth** [reply]
Ya, doing that. Creating a new repo to see if I can replicate it for you.

**Reply by joshhanley** [reply]
@abishekrsrikaanth thanks! Much appreciated 🙂

**Reply by abishekrsrikaanth** [reply]
@joshhanley i am going to close this as I created a new laravel project and copied over all my code and there seems to be no issues. I am still not quite sure what the issue is with the other project. Thank you for your time.

**Comment by onensensy** [comment]
I am having the same issue, I cannot really start from a fresh laravel project, What can i do

**Comment by jonerickson** [comment]
Receiving this error as well. Any help?

**Reply by onensensy** [reply]
I noticed the error came when I used Livewire's Layout. Try using the default layout 'app-layout' that comes when you scaffold breeze or jetstream. That worked for me

**Reply by jonerickson** [reply]
I am using the jet stream scaffolding. Are you injecting the live wire assets? Doesn't seem to make a difference for me.

**Comment by jonerickson** [comment]
If anyone struggling with this error, I found the cause of mine - simply just an extra space at the top of one of my PHP files. The opening PHP tag started on line 2 instead of 1.
